src/main/java/com/google/cloud/resourcemanager/ResourceManager.java

```diff
@@ -161,6 +161,36 @@ public static ProjectListOption fields(ProjectField... fields) {
     }
   }
 
+  /** Class for specifying Lien list options. */
+  class LienListOption extends Option {
```

Comment:
can this be a non-inner class? That would make it easier to read.

Reply:
correct but this class used only one place so i would like to kept it, as it is.

Reply:
True, but I don't see why that implies it should be an inner class. IMHO inner classes are way overused. They make code slightly faster to write, but encourage violation of encapsulation in a way that makes it harder to read.

Reply:
done